examples/restaurantbot/restaurant_domain.yml

```diff
@@ -51,7 +51,7 @@ templates:
   utter_ask_location:
     - "where?"
   utter_ask_moreupdates: 
-    - "if you'd like to modify anything else, please tell me what"
+    - "Price ranga={price} Cuisine={cuisine} Location={location} Count={people}. Would you'd like to modify anything else, please tell me what?"
```

Comment:
There's a typo in "price range". Also I'd replace the "would" with "if" and leave the question mark

Reply:
Previous change would take care of it..